bukkit/src/main/java/org/popcraft/chunky/platform/BukkitWorld.java

```diff
@@ -53,6 +53,10 @@ public String getKey() {
         return world.getKey().toString();
     }
 
+    public org.bukkit.World getBukkitWorld() {
```

Comment:
I'd like to keep the abstraction clean here if possible, which it should be based on my suggestion in your other PR, by adjusting the implementation of `getElevation` here (which already has access to the underlying world).